docs/adr/0010-orchestrator-architecture-review.md

```diff
@@ -0,0 +1,80 @@
+# ADR-0010: Multi-Room Orchestrator Architecture Review
```

Comment:
we have 2 adr docs. one looks like the proposal and the other the accepted one. we only need 1 adr record per decision